.github/workflows/documentation_accessibility_checks.yml

```diff
@@ -0,0 +1,90 @@
+name: '[Documentation] Accessibility Check'
+
+# Adapted from https://docusaurus.io/docs/deployment
+
+on:
+  workflow_dispatch:
```

Comment:
Can we add a branch option here? There's an example of this in the on demand build workflow

Reply:
I think we'll get this "for free" once the workflow has been merged to `develop` at least once. Additional comments on this below regarding the site map.